doc/code_coverage_rate/code_coverage_rate.md

```diff
@@ -0,0 +1,173 @@
+#  [Code Coverage Rate HLD] Update code coverage rate.md
+code-coverage-rate-hld
+
+# About this Document
+This document provides a high level of design for code coverage rate testing.
+
+# Problem Definition
+Code coverage rate is a term that describes the rate of the number of code lines that have been executed divide by the number of total code lines in the switch operating system. We reuse this term in SONiC code testing and create some related tools for SONiC code testing.
```

Comment:
Shouldn't this be _code coverage ***percentage***_ instead of _code coverage ***rate***_  ?

Reply:
"percentage" might be a more accurate word to use since the degree of code coverage is represented by "%" in both GCOV and Coverage. But the word "rate" can also be found in many articles talking about code coverage. So we consider it doesn't matter to use which of one if both of these two words are using right now.